daisy_workflows/image_build/enterprise_linux/rhel_9_byos_arm64.wf.json

```diff
@@ -0,0 +1,57 @@
+{
+  "Name": "build-rhel-9-arm64",
+  "Vars": {
+    "google_cloud_repo": {
+      "Value": "stable",
+      "Description": "The Google Cloud Repo branch to use."
+    },
+    "installer_iso": {
+      "Required": true,
+      "Description": "The RHEL 9 installer ISO to build from."
+    },
+    "build_date": {
+      "Value": "${TIMESTAMP}",
+      "Description": "Build datestamp used to version the image."
+    },
+    "publish_project": {
+      "Value": "${PROJECT}",
+      "Description": "A project to publish the resulting image to."
+    }
+  },
+  "Steps": {
+    "build-rhel": {
+      "Timeout": "60m",
+      "IncludeWorkflow": {
+        "Path": "./enterprise_linux.wf.json",
+        "Vars": {
+          "el_release": "rhel-9",
+          "kickstart_config": "./kickstart/rhel_9_byos_arm64.cfg",
+          "machine_type": "t2a-standard-4",
+          "google_cloud_repo": "${google_cloud_repo}",
+          "installer_iso": "${installer_iso}",
+          "rhel_byos": "true"
+        }
+      }
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "rhel-9-byos-arm64-v${build_date}",
+          "SourceDisk": "el-install-disk",
+          "Licenses": [
+            "projects/rhel-cloud/global/licenses/rhel-9-byos"
+          ],
+          "Description": "Red Hat, Red Hat Enterprise Linux BYOS, 9, aarch64 built on ${build_date}",
+          "Family": "rhel-9-byos-arm64",
+          "GuestOsFeatures": ["UEFI_COMPATIBLE", "VIRTIO_SCSI_MULTIQUEUE", "SEV_CAPABLE"],
```

Comment:
No GVNIC?

Reply:
Fixed, but I'll make sure all of the workflows match the real publish workflows in terms of guestOSFeatures.